src/aspire/source/image.py

```diff
@@ -145,7 +145,7 @@ def states(self, values):
 
     @property
     def filter_indices(self):
-        return self.get_metadata("__filter_indices")
+        return np.atleast_1d(self.get_metadata("__filter_indices"))
```

Comment:
I'm not following entirely here. Under what circumstances is this not 1D? I thought the idea was to get rid of `np.squeeze`?

Reply:
I think `squeeze` is fine, it just breaks down when the metadata field has one row and one column. That is, the metadata value is a scalar, and we only have one image in the `Source`. What happens is when all axes are 1-dimensional, `squeeze` returns a 0-D array (e.g. `array(0)`) when we actually want a 1-D array (e.g. `array([0])`) which is subscriptable.

This also occurs with these two `ImageSource` properties that are single-column. 

https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/3223d6c2c308af66f321ca2b0ddf3933045f904b/src/aspire/source/image.py#L132-L134

https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/3223d6c2c308af66f321ca2b0ddf3933045f904b/src/aspire/source/image.py#L187-L189

Reply:
Yes but instead of fixing the output of `get_metadata` everywhere, it would be better to fix `get_metadata` once and for all. Otherwise I have a feeling that the same bug will crop up elsewhere.

Reply:
#564 created

Reply:
👍